templates/email/order-cancel-failed.hypr

```diff
@@ -0,0 +1,22 @@
+﻿{% extends "email/email" %}
+{% block body-content %}
+<!--- Header --->
+<div class="mz-b2b-email-header">  
+  <div class="header-logo">
+    <img src="https://kibocommerce.com/wp-content/themes/kibo/images/logo.png" alt="logo">
+  </div>  
+  <div class="header-text">
+    <div class="store-name">{{ siteContext.generalSettings.websiteName }}</div>
+  </div>
+</div>
+<br />
+<div>
+    <br /><br />
+    {{ labels.orderCancelFailed|string_format(model.orderNumber)|safe }} 
+     <a href="//{{ adminDomainName }}/Admin/s-{{model.siteId}}/orders/edit/{{model.id}}">https://{{ adminDomainName }}/Admin/s-{{model.siteId}}/orders/edit/{{model.id}}</a>.
```

Comment:
Need to add `https` to the `a href` not just to the text.

Reply:
Done.